### DISCUSSION TITLE
No Wi-Fi - want consumption in AP mode

### DISCUSSION THREAD

**Discussion by marsrakete** [arendst/Tasmota#19079] [Q&A] [answered]
Nous Tasmota A1T

Sorry if answered before, couldn’t find a solution. I have no wifi in my garage but want to protocol power generation of my solar modules on the garage roof, connected to the Nous. How can I do that? Accessing the AP mode on my phone works, but the main menu is missing. As the connection breaks when trying to access the console with the browser, I have no clue so far.

**Comment by barbudor** [comment, marked as answer]
Tasmota is not designed to work in AP mode. AP mode is exclusively for initial configuration. 
You have to connect to an AP for normal operation and provide a NTP server (or an access to the Internet)


Note that there are some discussion around about modifying Tasmota to make it work in AP mode but this is not supported/endorsed by the developers team for now.

**Reply by marsrakete** [reply]
Thank you, I thought that much, hoped for more :-)

**Comment by Noschvie** [comment]
What about using a WLAN Powerline Adapter ?

**Reply by marsrakete** [reply]
That is a costly but probably the only useful solution.

**Reply by sfromis** [reply]
Depending on how the distances map out, sometimes a cheap strategy could be to deploy Tasmota on a dev board as a low-cost range extender, similar to a Wifi repeater. This seems to work well, I've tested it with web browsing on a tablet.

**Reply by joba-1** [reply]
I use the tasmota range extender for months now. More stable than my router :)
